File: lua/hello-world/hello-world.lua
--
-- Solution file for the Lua "Hello World" problem
-- Written by Sofia Nieves
-- Thu 26 May 16:52:55 UTC 2016
--

local hello_world = {}

function hello_world.hello(name)
  if name ~= nil then
    return ('Hello, ' .. name .. '!')
  end
  return 'Hello, world!'
end

return hello_world

File: go/hello-world/hello_world.go
//
// Completed by Sofia Nieves
// Mon 30 May 15:26:37 UTC 2016
//
package hello

const testVersion = 2

// It's "Hello world"
func HelloWorld(name string) string {
	if name != "" {
		return "Hello, " + name + "!"
	}
	return "Hello, World!"
}
